src/modules/session/thunks.ts

```diff
@@ -113,28 +116,74 @@ export const updateInitialProfile = createAsyncThunk(
       return;
     }
 
-    history.push("/home");
+    history.push("/home/profile");
   }
 );
 
 /**
- * Fetch inital data. If no profile information is present,
- * navigate to the beginning of the edit profile form.
+ * Logs in and navigates to the library page.
  */
-export const getInitialData = createAsyncThunk(
-  "session/getInitialData",
-  async (_, { dispatch }) => {
-    const profileResponse = await dispatch(
-      sessionApi.endpoints.getProfile.initiate()
+export const login = createAsyncThunk(
+  "session/login",
+  async (body: LoginRequest, { dispatch }) => {
+    const loginResponse = await dispatch(
+      sessionApi.endpoints.login.initiate(body)
     );
 
-    if ("error" in profileResponse) {
-      return;
-    }
+    if ("error" in loginResponse) return;
 
-    if (!profileResponse?.data?.firstName) {
-      history.push("/create-profile/what-is-your-first-name");
-    }
+    history.push("/home/upload-song");
```

Comment:
You mentioned in the PR description: 
> * Navigates to ... library page on subsequent logins

Was that note an error or does this `history.push` need to be updated along with the socials below?

Reply:
Saw the Slack convo just needs an update to the PR description.